index.html

```diff
@@ -678,6 +678,193 @@ <h3>JsonWebSignature2020</h3>
           </pre>
       </section>
 
+      <section id="BbsBlsSignature2020" about="https://w3id.org/security#BbsBlsSignature2020"
+      typeof="rdfs:Class">
+
+        <h3>BbsBlsSignature2020</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which deterministically names all
+unnamed nodes. Importantly, a BbsBlsSignature digests each of the statements
+produced by the normalization process individually to enable selective disclosure. 
+The signature mechanism uses Blake2B as the digest for each statement and produces a single 
+output digital signature.
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">unstable</dd>
+            <dt>Parent Class</dt>
+            <dd>Signature</dd>
+            <dt>Expected properties</dt>
+            <dd>verificationMethod, proofValue</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/rdf#URDNA2015">
+https://w3id.org/rdf#URDNA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="https://mattrglobal.github.io/bbs-signatures-spec/#name-sign">
+  https://mattrglobal.github.io/bbs-signatures-spec/#name-sign</a>
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </ul>
+        <p>
+A Bbs signature is a cryptographic digital signature that works with any pairing friendly elliptic
+curve, a BbsBls signature is a BBS signature using keys from the BLS12-381 curve:
+        </p>
+          <p>
+The example below shows how a basic JSON-LD proof is expressed in a
+JSON-LD snippet. Note that the proof property is directly embedded in the
+object. The signature algorithm specifies how the proof can be generated
+and verified.
+          </p>
+          <pre class="example prettyprint language-jsonld">
+{
+  "@context": ["https://w3id.org/security/v1", "http://json-ld.org/contexts/person.jsonld"]
+  "@type": "Person",
+  "name": "Dave Longley",
```

Comment:
nice

Reply:
:) just following the pattern @dlongley let me know if you would rather not be the example!